api/src/main/java/io/onetable/model/sync/SyncResult.java

```diff
@@ -53,6 +53,8 @@ public enum SyncStatusCode {
   @Value
   @Builder
   public static class SyncStatus {
+    public static SyncStatus SUCCESS =
```

Comment:
probably better to do for failure as well ?

Reply:
Are you thinking of something like `SyncStatus.forFailure(String errorMessage, String errorDescription, boolean canRetry)`?

Reply:
I think no much gain vs Sucess object where we have singleton object.